platform/mds/mds/src/main/java/org/motechproject/mds/config/MdsConfig.java

```diff
@@ -110,13 +115,17 @@ public Properties getDataNucleusPropertiesForInternalInfrastructure() {
         properties.remove("javax.jdo.option.Optimistic");
         properties.remove("datanucleus.flush.mode");
         addBeanValidationFactoryProperty(properties);
+        properties.put("javax.jdo.option.ConnectionURL", getSchemaConnectionURL(properties));
```

Comment:
use the constant defined earlier (moreover, aren't jdo connection options available from some sort of options class)